pyccel/errors/messages.py

```diff
@@ -123,6 +123,10 @@
 INVALID_FILE_EXTENSION = 'Wrong file extension. Expecting `py` of `pyh`, but found'
 INVALID_PYTHON_SYNTAX = 'Python syntax error'
 
+# ARRAY ERRORS
+ASSIGN_ARRAYS_ONE_ANOTHER = 'assigning arrays one another NOT IMPLEMENTED'
```

Comment:
There seems to be a word missing here. Can I suggest:
```
Arrays which own their data cannot become views on other arrays
```
Does this seem clear to you? Does it correctly describe the situation?

Reply:
yeah I think it describe it more clearly